pkg/types/baremetal/defaults/platform.go

```diff
@@ -35,4 +40,15 @@ func SetPlatformDefaults(p *baremetal.Platform) {
 			host.HardwareProfile = HardwareProfile
 		}
 	}
+
+	if p.ApiVIP == ApiVIP {
```

Comment:
nit: `if p.ApiVIP == ""` would be clearer to me what's going on

Reply:
I changed it to use the const because it is also referenced in the baremetal installconfig survey validation and I wanted to make sure the values matched, even if we changed the default for some reason. I do agree that referencing it from within the package is confusing though, so I'd be okay changing it back.

Reply:
Oh I see, that makes sense to me, it's fine then